### DISCUSSION TITLE
how to make my integrated_title_buttons transparent and look normal

### DISCUSSION THREAD

**Discussion by Joey901201** [wezterm/wezterm#5459] [Q&A]
### What Operating System(s) are you running on?

Windows

### WezTerm version

20240203-110809-5046fc22

### Ask your question!

![image](https://github.com/wez/wezterm/assets/61229399/22d15d87-da74-4e6c-b72e-c612ad68871b)
is there a way I can make the integrated_title_buttons also transparent?

I've tried this and it didn't work, 
`integrated_title_button_color = "rgba(30, 30, 46, 0.6)"`

and also the button looks unusual, is there a way to make it look normal?
here's my config rn: 
```
window_background_opacity = 0.6,
color_scheme = "Catppuccino Mocha",
use_fancy_tab_bar = false,
window_decorations = "INTEGRATED_BUTTONS|RESIZE",
integrated_title_button_style = "Windows",
integrated_title_button_color = "auto",
integrated_title_button_alignment = "Right",
```

and I wish the button to look like my vscode:
![image](https://github.com/wez/wezterm/assets/61229399/f0b6be25-5754-42a8-b763-0bc86dbc5ca9)

**Comment by httpsrichard** [comment]
Figured it out finally. Retro integrated buttons inherit `new_tab` values

```lua
config.colors                    = {
    tab_bar = {
        new_tab = {  -- shared with retro bar integrated window buttons
            bg_color = 'red',
            fg_color = '#808080',
        },
        }
```
